ros/launch/odometry.launch.py

```diff
@@ -42,7 +42,7 @@ def generate_launch_description():
             DeclareLaunchArgument("bagfile", default_value=""),
             DeclareLaunchArgument("visualize", default_value="true"),
             DeclareLaunchArgument("odom_frame", default_value="odom"),
-            DeclareLaunchArgument("base_frame", default_value="base_footprint"),
+            DeclareLaunchArgument("base_frame", default_value=""),
```

Comment:
I'm unsure which would be the best sane "default" for this one.  I guess most ROS applications would already have a `base_link` or `base_footprint` defined in the tf tree.

@tizianoGuadagnino any comments?

Reply:
If we leave this parameter empty by default, then at each iteration `egocentric_estimation` will evaluate to True they way it's written now. As a result, the local map and trajectory will be reported in the sensor frame instead of the base frame, regardless of the specifics of the user's TF tree. If we set the default argument to `base_link`, then I think that attempting to compute `pose` will fail if  `base_link` is not connected to the frame of the input PointCloud2 in the user's TF tree:

```
[ERROR] [1697646211.202267521]: Failed to find tf between base_link and crl_rzr/base. Reason=Could not find a connection between 'velodyne-link' and 'base_link' because they are not part of the same tree.Tf has two or more unconnected trees.
```

Therefore, to me, leaving `base_frame` empty seems like a less limiting option.

Reply:
Ok, I agree on this one. Will replicate also this on the ROS 2 node